schema.sql

```diff
@@ -1,5 +1,3 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animal (id BIGSERIAL, name VARCHAR(50), date_of_birth DATE, escape_attempts INT, neutered BOOL, weight_kg DECIMAL);
```

Comment:
- [ ] [OPTIONAL] Good job making the id column autoincremented, just one suggestion you can add the primary constraint and use the ID coulmn as Primary key :+1:

Reply:
Hello, @PinkMoon25 thank you for this out for me